hip-clang.docker

```diff
@@ -67,3 +67,15 @@ RUN export HIPCC_COMPILE_FLAGS_APPEND='-O3 -Wno-format-nonliteral -parallel-jobs
 # install last released miopentensile in default, install latest commits when MIOTENSILE_VER="latest"
 RUN if [ "$MIOTENSILE_VER" = "latest" ] ; then cget -p $PREFIX install ROCmSoftwarePlatform/MIOpenTensile@7568654c938d42e9a91c6b18fb382f5b978d12fd; else cget -p $PREFIX install ROCmSoftwarePlatform/MIOpenTensile@5fe0bf4a8dc59f3ab62df929297280915372ce16; fi
 
+RUN cd ~ && \
+    git clone -b miopen-dialect https://github.com/ROCmSoftwarePlatform/llvm-project-mlir.git && \
```

Comment:
This should download the source tarball as cloning is much slower. The source tarball is here:

https://github.com/ROCmSoftwarePlatform/llvm-project-mlir/archive/5b26e1305d6ba82c385f560287f6af97057c7642.tar.gz

Reply:
Nice catch, thanks!